src/potential/electrostatics/electrostatics.h

```diff
@@ -30,14 +30,16 @@ class Electrostatics {
                     std::vector<double> &pol, std::vector<double> &sys_xyz, std::vector<std::string> &mon_id,
                     std::vector<size_t> &sites, std::vector<size_t> &first_ind,
                     std::vector<std::pair<std::string, size_t> > &mon_type_count, bool do_grads = true,
-                    double tolerance = 1E-16, size_t maxit = 100, std::string dip_method = "iter");
+                    double tolerance = 1E-16, size_t maxit = 100, std::string dip_method = "iter",
+                    std::vector<double> box = {1000.0, 0.0, 0.0, 0.0, 1000.0, 0.0, 0.0, 0.0, 1000.0},
```

Comment:
Any user-facing documentation should specify whether the box is stored in column-major or row-major order, if it doesn't already.  There is no difference for orthorhombic boxes, of course.

Reply:
The box is specified in system.cpp/h, where I say how is it supposed to be. In principle, users should not use directly the electrostatics class. In any case, this is something planned to do when I go over documentation of eletrostatics (i don't wanna change source code here to avoid conflicts when you modify something, specially if I start to change function names and add a bunch of comments and documentation.) I assum that `box = {v1x,v1y,v1z,v2x,v2y,v2z,v3x,v3y,v3z}` where v1, v2, v3 are the three lattice vectors of the box.